app/views/pages/about.html.haml

```diff
@@ -43,9 +44,12 @@
           Purpose
         %p.text-p-2xlarge
           Exercism exists to help as many people as possible
+          - # TODO
```

Comment:
@ErikSchierboom `using/concepts/fluency` - we have a doc in website-copy or legacy docs on this. (And better word than `concepts`?)